crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -92,6 +113,7 @@ uint64_t FIPS_service_indicator_after_call(void) { return 0; }
 int FIPS_service_indicator_check_approved(int before, int after) { return AWSLC_NOT_APPROVED; }
```

Comment:
If we are not in FIPS mode should we be returning approved for everything instead?  Wouldn't want to impact a random application with incorrect FIPS checks in their code..

Reply:
The current approach we've taken this is returning `AWSLC_APPROVED` for the wrapper macro we're advocating that people use, and `AWSLC_NOT_APPROVED` for the actual approval check. I do see how this might be a bit confusing however.. Should I change the function to align with the macro?
Macro: https://github.com/awslabs/aws-lc/blob/main/include/openssl/service_indicator.h#L52-L59
Current Test: https://github.com/awslabs/aws-lc/blob/main/crypto/fipsmodule/service_indicator/service_indicator_test.cc#L205-L219

Reply:
I'm fine either way as long as it's documented well enough so I can choose my own adventure to find the right answer :)  Adding a few comments would make me happy.

Reply:
Added description in `service_indicator.h`.